packages/nimble-components/src/accordion/specs/IxD.md

```diff
@@ -0,0 +1,108 @@
+-   To shorten scrolled views when not all information is equally important
+-   In views with reduced horizontal or vertical space
+
+Do not use an accordion:
+-   For a single panel of content
+-   To organize unrelated information into panels
+-   For selectable or navigable lists or trees
+
+Best practice is to use multiple disclosures in one accordion and avoid one disclosure in multiple accordions.
+
+Do this:  
+![ ](spec-images/Accordion-13.png)
+
+Don't do this:  
+![ ](spec-images/Accordion-14.png)
+
+Multiple disclosures can be used in a row for panels of unrelated content, at the client user's discretion and in accordance with component guidelines and product consistency.
+
+Avoid this:  
+![ ](spec-images/Accordion-15.png)
+
+If only one section (relative to the content) is needed, use a disclosure component.  
+![ ](spec-images/Accordion-16.png)
+
+### Tree & Navigation
+With a disclosure/accordion, the heading is a label for a group of content, not content in-and-of itself. With a tree, the content is the list items and all the list items represent data.
+
+| Component  | Content    | Organization |
+| ---------- | ---------- | ------------ |
+| Disclosure | Any        | Group labels |
+| Tree       | List items | Parent/child |
+
+[ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/treeview/examples/treeview-navigation/) recommends avoiding using a tree component for navigation because the `tree` role requires unecessary implementation of complex functionality that is not needed for navigation styled to look like a tree with expandable sections. However, the accordion is not necessarily the correct component to try and replace it with.
+
+| Component  | Content    | Organization |
+| ---------- | ---------- | ------------ |
+| Disclosure | Any        | Group labels |
+| Tree       | List items | Hierarchical |
+| ???        | List items | Group labels |
+
+[**QUESTION:** Maybe a new navigation-specific component?]
+
+## Design
+### Interactions & States
+Header buttons are used to expand and collapse associated panels of content. The client user can specify which panels (if any) are open by default.
+
+![ ](spec-images/Accordion-1.png)
+
+Hovering over a header button indicates interactivity:  
+![ ](spec-images/Accordion-2.png)
+
+Clicking anywhere on a header button opens (if closed) or closes (if opened) the associated panel of content. [**QUESTION:** Do panels only display block content (one component per "line") or do they display flex content?]  
+![ ](spec-images/Accordion-3.png)
+
+The client user can specify whether one panel or more than one panel can be opened at a time. If the former, clicking to open a closed panel will close the open panel (if applicable).  
+![ ](spec-images/Accordion-4.png)
+
+If a component in a panel is in an error state, the error state is visually indicated on the associated header button when closed (and maybe also be visually indicated when open).  
```

Comment:
This point may be worth clarification from a technical perspective. I presume the desired behavior is that if there is any component within an accordion-item/disclosure that has an error-state (the technical meaning of is that a component has an `error-visible` attribute set), that the accordion-item itself would visually represent that error at the very least when it is collapsed.

If this is, in fact, how we want the component to behave (i.e. requires no user configuration), then there are technical implications to that, namely that we will be putting a burden on the component to track DOM changes to _all_ of its children.

The alternative to this is that we instead create a user-facing API for the accordion-item itself that allows a user to specify that it's in an error state (i.e. the `error-visible` attribute).

Personally, I would like the `accordion-item` to behave as described here, however I'm not sure if we should vet the performance implications of this.